pkg/cnab/config-adapter/stamp.go

```diff
@@ -109,8 +114,9 @@ func (c *ManifestConverter) DigestManifest() (string, error) {
 	v := pkg.Version
 	data = append(data, v...)
 
-	for _, m := range c.Mixins {
-		data = append(append(data, m.Name...), m.Version...)
+	usedMixinsVersion := c.getUsedMixinsVersion()
+	for usedMixinName, usedMixinVersion := range usedMixinsVersion {
+		data = append(append(data, usedMixinName...), usedMixinVersion...)
```

Comment:
I think the way we are concatenating these bytes is already pretty odd but I don't want to have you try to fix that in this PR since it's unrelated. Once your PR is in, I'll submit a PR to clean that up.

Reply:
It's just hard to read and when thinking about the digest as a whole, hard to follow what we are hashing here.